bin/cfg/Tomb1Main_gameflow.json5

```diff
@@ -363,6 +363,8 @@
                 {"type": "display_picture", "picture_path": "data\\cred1.pcx", "display_time": 7.5},
                 {"type": "display_picture", "picture_path": "data\\cred2.pcx", "display_time": 7.5},
                 {"type": "display_picture", "picture_path": "data\\cred3.pcx", "display_time": 7.5},
+                // {"type": "display_picture", "picture_path": "data\\install.pcx", "display_time": 7.5},
+                {"type": "total_stats"},
```

Comment:
Let's not hardcode the path to `install.pcx` and instead use it as an argument here similar to `GFS_DISPLAY_PICTURE`. Remember to free the memory for the argument in `gameflow.c`.

Reply:
Ok. I wasn't sure if we wanted to leave it up to the builders to display the stats over a picture, an in game frame, or a custscene.